### DISCUSSION TITLE
Oidc issue

### DISCUSSION THREAD

**Discussion by Oneel77** [seriousm4x/UpSnap#664] [Help wanted]
Hi,
I was able to create an OIDC Connect provider in UpSnap. Authentication works fine.

The first user is created at login as expected but not the second one.
I have an error "fail to authenticate" with the following details:
- The request requires valid record authorization token to be set.
- constraint failed: UNIQUE constraint failed: _externalAuths.collectionId, _externalAuths.recordId, _externalAuths.provider (2067)

If I delete the first user I created, I'm able to login with the second user for which it doesn't work previously.

Thanks for the support

**Comment by seriousm4x** [comment]
Hi, i didn't implement OIDC support myself. It's a feature of [pocketbase](https://github.com/pocketbase/pocketbase/).
Please try with the [latest pocketbase binary](https://github.com/pocketbase/pocketbase/releases) and if the issue persists, [create an issue](https://github.com/pocketbase/pocketbase/issues) there. Thanks
